spec/views/users/users_index_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'User', type: :feature, js: false do
+  before do
+    # Create some sample users for testing with unique email addresses
+    user1 = User.create(name: 'John', email: 'john@example.com', photo: 'https://example.com/john.jpg')
+    user2 = User.create(name: 'Alice', email: 'alice@example.com', photo: 'https://example.com/alice.jpg')
+    Post.create(title: 'Post 1', text: 'Content 1', author_id: user1.id)
+    Post.create(title: 'Post 2', text: 'Content 2', author_id: user1.id)
+    Post.create(title: 'Post 3', text: 'Content 3', author_id: user2.id)
+  end
+
+  it 'displays the list of users' do
+    visit '/users'
+
+    expect(page).to have_content('John')
+    expect(page).to have_content('Alice')
+  end
```

Comment:
- [ ] The error message indicates that there is an unknown attribute 'email' for the User model. This means that the User model in your Rails application does not have an 'email' attribute defined. If you check your schema you will see that indeed there is no "email" field for the user table.

![image](https://github.com/HERMON-1995/Blog_app/assets/86318284/4f1657fc-432a-4fe9-8553-2c527707fea1)